inventories/demo2/group_vars/commcare_sync/vars.yml

```diff
@@ -0,0 +1,10 @@
+---
+
+public_host: commcare-sync-demo2.dimagi.com  # 35.174.183.59
```

Comment:
Intentionally uses a different name to avoid users trying to log into this instance. Once required changes have been made to CC Sync, we can change the name back, and maybe restore the original users.